pages/creating-microservices.md

```diff
@@ -11,17 +11,35 @@ sitemap:
 
 Microservices are a type of JHipster application, that have no front-end (the Angular front-end must be generated on a [gateway]({{ site.url }}/api-gateway/)), and which work with the [JHipster Registry]({{ site.url }}/jhipster-registry/) to be configured, discovered, and managed.
 
-<h2 id="generating_entities">Generating entities in a microservices architecture</h2>
+<h2 id="entities">Entities in a microservices architecture</h2>
+
+The user entity is not accessible in the microservice when using JWT or DTO, only when OAuth2 is use, there is a mechanism in the microservice to extract user data from the token and save it to the microservice's database. So for JWT and DTO you can not use and/or define a relationship with the User entity because it exists only in the gateway database.
+
+Relationships between entities from different microservices are not supported.
+
+<h2 id="generating_entities">Generating entities</h2>
+
+For entities in microfrontend see [Microfrontends](#microfrontends)
 
 Using the [entity sub-generator]({{ site.url }}/creating-an-entity/) works a little bit differently in a microservices architecture, as the front-end and the back-end codes are not located in the same application.
 
-First, generate the entities in the microservices applications: this works as usual, and you can also use [JHipster UML]({{ site.url }}/jhipster-uml/) or [JDL Studio](https://start.jhipster.tech/jdl-studio/) to help you generate complex entities and relationships. As microservices don't have a front-end, no Angular/React code will be generated. The user entity is not accessible in the microservice when using JWT or DTO, only when OAuth2 is use, there is a mechanism in the microservice to extract user data from the token and save it to the microservice's database. So for JWT and DTO you can not use and/or define a relationship with the User entity because it exists only in the gateway database.
+First, generate the entities in the microservices applications: this works as usual, and you can also use [JHipster UML]({{ site.url }}/jhipster-uml/) or [JDL Studio](https://start.jhipster.tech/jdl-studio/) to help you generate complex entities and relationships. As microservices don't have a front-end, no Angular/React code will be generated.
 
 Then, on the gateway(s), run the entity sub-generator again. A new question will appear at the beginning, which is specific to gateways:
 
 - You will have the choice either to generate a new entity normally (a gateway is also a standard JHipster application, so this would work like for a monolith application), or use an existing JHipster configuration from a microservice.
 - If you choose to generate the entity from a microservice, you will need to enter the path to this microservice on your local computer, and then JHipster will generate the front-end code on the gateway.
 
+<h2 id="microfrontends">Microfrontends</h2>
```

Comment:
```suggestion
## Microfrontends
```

The `<h2>` syntax is only needed when you want to change the anchor link (set by the `id` attribute).